src/components/Tag.tsx

```diff
@@ -0,0 +1,36 @@
+import PropTypes from "prop-types";
+import classnames from "classnames";
+import React from "react";
+
+import "govuk-frontend/govuk/components/tag/_tag.scss";
+
+/**
+ * The proptypes for {@link Tag}.
+ */
+export interface TagProps {
+  id?: string;
+  className?: string;
+  children: React.ReactNode;
+}
+
+/**
+ * A component for indicating the status of something, such as
+ * an item on a task list or a phase banner.
+ */
+export const Tag: React.FunctionComponent<TagProps> = ({
+  id,
+  className,
+  children
+}): JSX.Element => {
```

Comment:
Pull the props destructuring into the body of the funtcion and give the argument a type, so the docs can document it (and regenerate the docs).